ambry-frontend/src/main/java/com/github/ambry/frontend/AmbrySecurityService.java

```diff
@@ -252,8 +252,10 @@ public void processResponse(RestRequest restRequest, RestResponseChannel respons
       }
     }
     if (quotaManager != null) {
-      quotaManager.charge(restRequest, blobInfo);
-      // TODO Use ThrottlingRecommendation returned from charge() to set appropriate response headers.
+      ThrottlingRecommendation throttlingRecommendation = quotaManager.charge(restRequest, blobInfo);
+      if (throttlingRecommendation != null) {
```

Comment:
@lightningrob As a followup from our discussion in previous PR, I feel that checking if requestQuotaThrottling is enabled is cleaner than checking for null throttlingRecommendation. Checking for null throttlingRecommendation makes us depend on the implementation of interface which is not clean. Let me know if you are ok to change
`if(quotaManager!=null)` to `if(quotaManager!=null && quotaManger.getQuotaConfig().isRequestThrottlingEnabled())`

Reply:
Seems fine.